### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/analytics.usa.gov#319]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)

## Results for https://analytics.usa.gov:
* __Error:__ This select element does not have a name available to an accessibility API. Valid names are: label element, title attribute.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
 * #agency-selector-wrap > select
 * `<select class="agency-selector"> <option value="/...</select>`

* __Error:__ This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
 * WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
 * #agency-selector-wrap > select
 * `<select class="agency-selector"> <option value="/...</select>`

* __Error:__ This select element does not have a name available to an accessibility API. Valid names are: label element, title attribute.
 * WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
 * #agency-selector-wrap-small > select
 * `<select class="agency-selector"> <option value="/...</select>`

* __Error:__ This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
 * WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
 * #agency-selector-wrap-small > select
 * `<select class="agency-selector"> <option value="/...</select>`

* __Error:__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.37:1. Recommendation: change background to #237fa0.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * #top_100_table > ul > li:nth-child(1) > a
 * `<a role="tab" data-type="Pages" aria-selected="true" href="#top-pages-realtime" class="site-nav">Now</a>`

## Summary:
* 5 Errors

**Comment by gbinal** [comment]
Should all be addressed by #322, so closing.  Thanks @nickbristow!
